tests/ltp/patches/ltp_fstatfs_fstatfs02.patch

```diff
@@ -0,0 +1,63 @@
+diff --git a/testcases/kernel/syscalls/fstatfs/fstatfs02.c b/testcases/kernel/syscalls/fstatfs/fstatfs02.c
+index db2230f82..385d846d0 100644
+--- a/testcases/kernel/syscalls/fstatfs/fstatfs02.c
++++ b/testcases/kernel/syscalls/fstatfs/fstatfs02.c
+@@ -20,6 +20,13 @@
+  * DESCRIPTION
+  *     Testcase to check fstatfs() sets errno correctly.
+  */
++/*
++ * Patch Description: Here one test is trying to access invalid address,
```

Comment:
Use single line (//) comments instead of block comments ( /* */ ).
Add patch description on top of the patch before “diff” line and add a single Line comment in code